subsys/bluetooth/controller/ll_sw/openisa/lll/lll.c

```diff
@@ -116,11 +116,9 @@ int lll_init(void)
 	ARG_UNUSED(clk_k32);
 
 	/* Get reference to entropy device */
-	dev_entropy =
-		device_get_binding(DT_LABEL(DT_INST(0, openisa_rv32m1_trng)));
+	dev_entropy = device_get_binding(DT_CHOSEN_ZEPHYR_ENTROPY_LABEL);
```

Comment:
@George-Stefan @alexandru-porosanu-nxp
You will have to decide if you rather want `device_get_binding(DT_LABEL(DT_INST(0, openisa_rv32m1_trng)));`
The way I have done it now this PR shouldn't change anything for ll_sw/openise apart from that the advertising random delay is now using the entropy device, (in master it now returns 0).
This is only based on my understanding of the code though, I haven't tested this since I don't have your HW.

Reply:
@joerchan , I'll test this later on tonight on our VEGA board and provide the results here